### PR TITLE
manifest: Update nrfxlib revision

### PR DESCRIPTION
**Pull request by rugeGerritsen**
Updates the SoftDevice Controller and MPSL.

Signed-off-by: Rubin Gerritsen <rubin.gerritsen@nordicsemi.no>

### CODE REVIEW

**Review by LundinJohan — APPROVED** [review]

**Comment by rugeGerritsen** [comment]
Rebased on main, now picking up a new push to nrfxlib

**Comment by DatGizmo** [comment]
Right now this PR runs all Integration tests because it changes sdk-nrfxlib in west.yml and there is no test-spec.yml in that repo yet.
You can add changes to this PR [nrfconnect/sdk-nrfxlib#687](https://github.com/nrfconnect/sdk-nrfxlib/pull/687) to make the test spec here more targeted

Then run CI here to update the test spec.

[Information about the test spec feature](https://projecttools.nordicsemi.no/confluence/display/NCS/Test+Spec)
[CI-Help for issues](https://ci-help.nordicsemi.no/)

**Comment by DatGizmo** [comment]
To get the manifest check to pass an update to the PR is needed for GitHub to use the correct .github/workflows/manifest.yml.

The best way to do this is to rebase the PR onto main, [so the fix is included.](https://github.com/nrfconnect/sdk-nrf/pull/7139)

A simple re trigger of the test sadly does not work as it will still use the faulty variant of the file.

File: west.yml [hunk 1]
```diff
@@ -108,7 +108,7 @@ manifest:
     - name: nrfxlib
       repo-path: sdk-nrfxlib
       path: nrfxlib
-      revision: f215460bec9869a2122059af28550792358a77e5
+      revision: pull/684/head
```
Comment (rlubos) [inline west.yml:111]: Please update SHA and rebase: `39a025f549ac7c3f9dc91c7c56415afe5f19e971`

**Review by rlubos — APPROVED** [review]

**Pull request merged**